packages/react/react/src/hooks/useLazyImage.ts

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { useRefEffect } from './useRefEffect';
+
+interface Props {
+  src: string;
+  threshold?: number | number[];
+  root?: Document | Element | null;
+  rootMargin?: string;
+  onInView?: () => void;
+}
```

Comment:
How about adding `onLoad` prop too?

Reply:
Is it correct that the onLoad function refers to the function that is executed when the image finishes loading?

Reply:
Exactly. But it can be subdivided:

- onLoadStart
- onLoadComplete

But I think it's not necessary now, it looks good to add only `onLoad`.

Reply:
I added "onLoadComplete" because "onLoad" can be confusing with "onLoad" in the img tag.